draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -333,6 +345,27 @@ The classic congestion control algorithm {{!RFC5681}} and the widely deployed
 Cubic algorithm {{?RFC9438}} do not address it, but a new
 congestion control algorithm has the opportunity to improve the state of the art.
 
+### Protection Against High Packet Loss
+
+A congestion control algorithm should reduce its sending
+rate if experiencing high packet loss.
+Even if a congestion control algorithm can tolerate high loss, many types
+of short request/response traffic do not want to pay the loss recovery
+performance penalty induced by high loss rates.
+
+The first version of the BBR algorithm {{BBRv1-draft}} failed this requirement.
+Experimental evaluation {{BBRv1-Evaluation}} showed that
+it caused a sustained rate of
+when multiple BBRv1 flows share a bottleneck and the buffer size was
+less than roughly one and a half BDP.
+This kind of behavior needed to be fixed, and indeed
+further versions of BBR {{BBR-draft}}
+fixed this issue.
```

Comment:
Neal gave me this "war story" when I was asking him for a specific example of what he had in mind -- see the first 2-3 comments in issue #101.

Part of the reason for including the description is to insist on the "sustained" nature of packet loss. Most algorithm end up causing a high rate of packet loss at the end of the startup period, but we don't want to say "don't use Reno because it causes lots of packet losses at the end of startup". Or "don't use Cubic because it causes some packet losses at the end of each epoch". BBRv1 is a neat example of the behavior that we find questionable.

Reply:
OK, I can live with it.